src/blocks/social-profiles/index.php

```diff
@@ -207,7 +207,7 @@ function coblocks_render_social_profiles_block($attributes)
 		esc_attr($text_align),
 		$markup
 	);
-
+	echo esc_attr($block_content);
```

Comment:
This bit is breaking the block. Need to return a string, not echo it. I'm guessing this was for debugging purposes.